sdcm/tester.py

```diff
@@ -443,7 +461,7 @@ def setUp(self):
                 self.legacy_init_nodes(db_cluster=self.db_cluster)
             else:
                 self.init_nodes(db_cluster=self.db_cluster)
-            if self.params.get('use_ldap_authorization') and not self.params.get('are_ldap_users_on_scylla'):
+            if (self.params.get('use_ldap_authorization') and not self.params.get('are_ldap_users_on_scylla')) or self.params.get('prepare_saslauthd'):
                 self.db_cluster.nodes[0].create_ldap_users_on_scylla()
                 self.params['are_ldap_users_on_scylla'] = True
             self.set_system_auth_rf()
```

Comment:
@amoskong , i think it was moved in a recent PR to be before the call to `create_ldap_users_on_scylla`

Reply:
Yes, the PR (https://github.com/scylladb/scylla-cluster-tests/pull/3428) has 'backport-2020.1 label.
I think someone will backport it later.

Reply:
I just applied your change suggestion, it's manually backport of commit 1d244d4bde956086d6ea093562c4b7fa258fb7f1

Reply:
thank you @amoskong